CHANGELOG.md

```diff
@@ -8,6 +8,14 @@
   Schemas.
   [#390](https://github.com/apiaryio/drafter/issues/390)
 
+* Drafter will now check the validity of header names and provide a warning
```

Comment:
It will also skip those headers with invalid chars. Don't know if that is needed to mention or not but previously it issued warning too but left the header intact.